code/_helpers/icons.dm

```diff
@@ -792,7 +791,7 @@ The _flatIcons list is a cache for generated icon files.
 	if(A.alpha < 255 && !islist(A.color))
 		flat.Blend(rgb(255, 255, 255, A.alpha), ICON_MULTIPLY)
 
-	return icon(flat, "", SOUTH)
+	return icon(flat, "", defdir)
```

Comment:
uhhhhh, i don't actually think this is correct. i'm *pretty* sure it should be south because south is the default state? whereas if you specify like, EAST, it'll have four states and all but EAST will be blank

Reply:
because we might want the single icon to be facing north/east/west instead of south